proto/one/operations/spreadsheet/computation_variable_binding.proto

```diff
@@ -23,4 +24,7 @@ message ComputationVariableBinding {
 
    // used to determine relative cell range for grouping functions
    string grouping = 8;
+
+   // The columnNumber of the reference column if different from columnNum
+   google.protobuf.UInt32Value referenceColumnNum = 9;
```

Comment:
Not sure why this is showing up as this was my change that was previously merged

Reply:
Looks like this is how it is in main and also my branch.